### ISSUE TITLE
Z-Wave Metering Dimmer does not properly implement Switch Level

### ISSUE THREAD

**Issue by andrewsayre** [SmartThingsCommunity/SmartThingsPublic#3986]
The [Z-Wave Metering Dimmer](https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy), and possibly others, do not implement `setLevel` as defined in the [SmartThings Capability Reference](https://smartthings.developer.samsung.com/docs/api-ref/capabilities.html#Switch-Level).  Specifically, they do not have an overload that accepts two numbers: `setLevel(level, rate)`.  While `rate` is not required, an overload should be provided that accepts the value and ignores it if not supported by the  device.

Using the new cloud API, there is no way of knowing whether the device supports `setLevel(level)` or `setLevel(level, rate)` and calling the latter results in an error because groovy can't find an overload to accept the parameters.

The fix is a simple adjustment from `setLevel(level)` to `setLevel(level, rate=0)` so Groovy generates the proper overloads.

Full list of DTHs that don't handle `setLevel` with two parameters (that I've found so far):
- GE Link Bulb
- Sylvania Ultra iQ
- ZigBee Accessory Dimmer
- ZigBee Dimmer Power
- ZigBee Dimmer with Motion Sensor
- ZigBee Dimmer
- ZLL Dimmer Bulb
- Z-Wave Metering Dimmer

**Comment by tpmanley** [comment]
Thanks for the bug report. We'll get this fixed.

**Comment by andrewsayre** [comment]
Thanks!  What kind of timeline are we looking at before it's updated and live?

**Comment by tpmanley** [comment]
I'm hoping to see it go out early next week. One thing to note is many of the DTHs with the issue have `executeCommandsLocally: true` which means that if the device is running locally on the hub the commands like `setLevel` are sent directly to the hub instead of the DTH. The hub does support the optional second parameter so the commands execute correctly in that case. Most of the errors I'm seeing are with DTHs that don't have `executeCommandsLocally: true`, namely `GE Link Bulb` and `LIFX Color Bulb`. We still plan to fix all of them but it does mean the impact is a little less than how it originally appears.

**Comment by andrewsayre** [comment]
I am finding users that have the Z-Wave Metering Dimmer, which is set to run locally, are affected and getting the error logged in the IDE about a missing method.

**Comment by andrewsayre** [comment]
Hmm I can’t find that handler.  Can you log in to the SmartThings IDE and confirm that’s the name of the “Type”?

**Comment by andrewsayre** [comment]
Hi Folks!  I see the PR was merged.  Do we know when this will go live?  Would love to get this issue (and the downstream issue) resolved! :)

**Comment by tpmanley** [comment]
@andrewsayre , sorry for the delay but this went live today. Can you take a look and let us know how it looks? Things are looking good on our end.

**Comment by andrewsayre** [comment]
Looks good!  Thanks!!

**Comment by Thegitbud** [comment]
Hi, I've recently installed Hassio and been running it for quite a while. I've integrated smartthings within the hassio integreations and it have been working well, without any issues. 

For some reason I had to use an old snapshot on the hassio to restore some features and then shit started to hit the fan. 

I had to re-configure the integration into hassio, no big deal. But when I tried to use the dimming feature this exact problem did occur wierdly enough. 

Im using the IKEA Bulbs with the ST handler: https://github.com/edvaldeysteinsson/IKEA-Tradfri/blob/master/devicetypes/edvaldeysteinsson/ikea-tradfri.src/ikea-tradfri.groovy

As this thread reads I reconfigured the handler accordingly but had to change setLevel(Value) to setLevel(Value, rate=0). 

This worked and the bulbs are dimmable again! 

BUT! 

Now the bulbs does only respond to changes in the brightness if I give it a two second resting period between brightness changes... 

This is what I can read from the ST Logs:
Smartthings Brightness Change:
https://pastecode.xyz/view/d08ab4d8

Hassio Brightness Change:
https://pastecode.xyz/view/58847822
